File: week1-c/mario/less/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    // Ask the user to enter a height
    int height;

    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    // Prints the pyramid until the specified height
    for (int i = 1; i <= height; i++)
    {
        // For loop to add spaces
        for (int k = 0; k < height - i; k++)
        {
            printf(" ");
        }

        // For loop to print the hashes
        for (int j = 0; j < i; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: week6-python/mario/less/mario.py
from cs50 import get_int

# Ask the user to enter a height
while True:
    height = get_int("Height: ")

    if height in range(1, 9):
        break

# Prints the pyramid until the specified height
for i in range(1, height + 1):

    # For loop to add spaces
    for k in range(height - i):
        print(" ", end="")

    # For loop to print the hashes
    for j in range(i):
        print("#", end="")

    print()
